src/levanter/models/llama.py

```diff
@@ -480,7 +486,7 @@ def init(cls, Vocab: Axis, config: LlamaConfig, *, key) -> "LlamaLMHeadModel":
         k_t, k_emb = jrandom.split(key, 2)
         transformer = LlamaTransformer.init(config, key=k_t)
         embeddings = LlamaEmbedding.init(Vocab, config, key=k_emb)
-        lm_head = hnn.Linear.init(In=config.Embed, Out=Vocab, key=k_emb, use_bias=config.use_bias)
+        lm_head = hnn.Linear.init(In=config.Embed, Out=Vocab, key=k_emb, use_bias=False, out_first=True)
```

Comment:
If we specify `out_first=True` at initializing lm_head (good to know such argument exists), do we still need the logic in `from_state_dict` and `update_state_dict`? I made those to revert the order in the weight.